vtex-apps/admin-example
# NAD-529 - Update template

NOTE: Most of the changes come from `yarn.lock` files.

**What problem is this solving?**

This boilerplate is outdated and the purpose of this PR is to update it with best practices, updated dev tools, better documentation with clearer instructions, the new standard UI library for the Admin - the admin-ui -, and a more secure way to handle i18n strings.

**How should this be manually tested?**

- Visit this workspace: https://marcelo--teamadmin.myvtex.com/admin/example and make sure everything is alright. 
- Go through the README.md and the jsDocs.
